bakery/src/tasks/pdfify-single.js

```diff
@@ -33,13 +32,11 @@ const task = (taskArgs) => {
         { name: fetchedInput },
         { name: fetchedResourcesInput },
         { name: styleInput },
-        { name: symlinkInput },
         { name: mathifiedInput }
       ],
       outputs: [{ name: artifactsOutput }],
       params: {
         ARTIFACTS_OUTPUT: artifactsOutput,
-        SYMLINK_INPUT: symlinkInput,
```

Comment:
I believe the PR is missing the corresponding update to `pdfify_single.sh` where symlinks are injected.

Reply:
And I can't believe that didn't cause an error when I ran through the steps!